src/strings/PowerFxResources.en-US.resx

```diff
@@ -4222,6 +4222,14 @@
     <value>Cannot reference virtual table {0} in formula columns.</value>
     <comment>Error Message. Used in formula columns for a formula that references a virtual table.</comment>
   </data>
+  <data name="FormulaColumns_TextNumberMissingFormat" xml:space="preserve">
```

Comment:
This is not the place where those strings should be... PFx assemblies should only have resources that are used by PFx. If this is used only by formula columns, then it should be on that project.

Reply:
This is being tracked by https://github.com/microsoft/Power-Fx-Dataverse/issues/192